### DISCUSSION TITLE
change config for one PWA

### DISCUSSION THREAD

**Discussion by lmamane** [Divested-Mobile/Mull-Fenix#234] [Q&A]
I have a PWA installed that is broken by the default `network.http.referer.XOriginPolicy=2`.

Is there a possibility to change that setting, but only for that PWA/URL/website? Similarly, it would be nice to enable/disable WebGL, WebAssembly, WebRTC, etc on a "as needed" basis rather than globally.

**Comment by SkewedZeppelin** [comment]
there is no built-in mechanism to change these for individual websites

you may consider setting it to 0 and try finding an extension that can do the same but only on some websites
extensions should work in the PWA view iirc
